examples/proc_FID_nutation.py

```diff
@@ -35,16 +37,28 @@
     fl.next("Raw Data with averaged scans", fig=fig)
     fig.suptitle("FID Nutation %s" % sys.argv[2])
     # }}}
-    # {{{ Apply overall zeroth order correction
-    s.ift("t2")
+    # {{{ Apply overall zeroth order correction, select the pathway, and apply a rudimentary alignment
     s /= prscr.zeroth_order_ph(
-        prscr.select_pathway(s["t2":0], s.get_prop("coherence_pathway"))
+        prscr.select_pathway(
+            s["t2":signal_range].sum("t2"), s.get_prop("coherence_pathway")
+        )
     )
+    s = prscr.select_pathway(
+        s["t2":signal_range], s.get_prop("coherence_pathway")
+    )
```

Comment:
b/c this gets done over and over below

Reply:
just select the signal of interest and move on

Reply:
I think the new rough_table_integrals takes care of this now